src/components/table/table-row/table-row.js

```diff
@@ -170,37 +196,37 @@ class TableRow extends React.Component {
   }
 
   /**
-   * @method componentWillUnmount
-   * @return {Void}
-   */
-  componentWillUnmount() {
-    if (this.context.detachFromTable) {
-      this.context.detachFromTable(this.rowID);
-    }
-  }
-
-  /**
    * @method componentWillReceiveProps
    * @return {Void}
    */
   componentWillReceiveProps(nextProps) {
-    if (this.props.uniqueID != nextProps.uniqueID) {
+    if (this.props.uniqueID !== nextProps.uniqueID) {
```

Comment:
Do we want to do this in every case - are there cases where `number !== string` but `number == string` and we need that to pass?